src/lib/queryAdvise.test.js

```diff
@@ -0,0 +1,11 @@
+import { getData } from './queryAdvise';
+import { renderIntoDocument } from '../../tests/utils/dom-render';
+
+describe('queryAdvise', () => {
+  describe('getData', () => {
```

Comment:
We should also make this file more flat

https://kentcdodds.com/blog/avoid-nesting-when-youre-testing